compiler/rustc_codegen_gcc/src/intrinsic/mod.rs

```diff
@@ -74,8 +74,14 @@ fn get_simple_intrinsic<'gcc, 'tcx>(
         sym::fabsf64 => "fabs",
         sym::minnumf32 => "fminf",
         sym::minnumf64 => "fmin",
+        sym::minimumf32 => "fminimumf",
+        sym::minimumf64 => "fminimum",
+        sym::minimumf128 => "fminimumf128",
```

Comment:
Thanks for switching to `fminimumf128`, but this is not going to work as is, though, since this calls `get_builtin_function` below.

You would need something like:
```rust
let f128_type = cx.type_f128();
return cx.context.new_function(
    None,
    FunctionType::Extern,
    f128_type,
    &[f128_type, f128_type],
    "fminimumf128",
    false,
);
```

Reply:
LLVM at least usually turns `f16` ops into `f32` versions wrapped in extend/trunc, any idea if GCC does something similar? If not then using the `*f16` functions should work.

LLVM doesn't emit calls to them at all currently, just some loose plans to do so in the future.

Reply:
I don't believe GCC does that.

Reply:
Okay, so let's leave the `f16` versions use the fallback body.

Reply:
Using the `*f16` functions definitely beats an ICE then, could be done as a follow up though. I've never actually tested how well the GCC backend supports f16 and f128.